include/conf/defaults.hpp

```diff
@@ -70,24 +72,34 @@ namespace color_coded
         "-isystem/opt/local/include",
         environment<env::tag>::clang_resource_dir, // internal libraries and intrinsics
         "-isystem/usr/include",
+#ifdef __APPLE__
         "-isystem/System/Library/Frameworks",
         "-isystem/Library/Frameworks",
+#endif
         "-w",
         "-fcolor-diagnostics" // See https://github.com/jeaye/color_coded/issues/104
       };
     }
 
+    /* Add system defaults to user specified arguments. Needed because libclang
+     * often fails to find system search paths. */
+    args_t add_defaults_to_args(std::string const &filetype, args_t &args)
```

Comment:
Woops, you do, due to `std::move`. Can we please either take an `args_t const &` and `std::copy` or take an `args_t &&`, modify, and return with an `std::move` (move in, move out). I don't think an output param is needed here.